test/services/formService.test.js

```diff
@@ -20,7 +21,7 @@ describe('formService', () => {
 
   beforeEach(() => {
     pdfFormatter = {
-      pickCurfewAddress: jest.fn().mockReturnValue(address),
+      pickCurfewAddress,
     }
```

Comment:
So that the function call is tested.  Exposes the bug. The test lower down shows that it is fixed.